packages/cubejs-client-react/index.d.ts

```diff
@@ -44,6 +44,31 @@ declare module '@cubejs-client/react' {
    */
   export const CubeProvider: React.FC<CubeProviderVariables>;
 
+  /**
+   * Cube.js context consumer
+   * ```js
+   * import React from 'react';
+   * import { CubeContext } from '@cubejs-client/react';
+   *
+   *
+   * export default function DisplayComponent() {
+   * const { cubejsApi } = React.useContext(CubeContext);
+   * const query = {
+   *   ...
+   * };
+   * const resultSet = await cubejsApi.load(query);
+   *   return (
+   *     <>
+   *       {resultSet.rawData().map(row => (
+   *         ...
+   *       ))}
+   *     </>
+   *   )
+   * }
+   * ```
+   */
+  export const CubeContext: Context<{ cubejsApi: CubejsApi }>;
```

Comment:
Better to factor the `type` out and refer to `Context` using `React.Context` (as all these types will go directly to our docs)

e.g.

```js
export const CubeContext: React.Context<CubeContextProps>
```

Reply:
followed `CubeProviderVariables` naming convention from above
so added `CubeContextVariables`